### DISCUSSION TITLE
Custom Quadrants

### DISCUSSION THREAD

**Discussion by CuriousTimmy** [rxhanson/RectanglePro-Community#246] [Q&A]
I'm a new user and I'm loving Rectangle Pro so far.

I have a couple of questions that I can't figure out. How can I create custom quadrants for key commands and snapping? I want to use the bottom row of my screen with 9ths and then fill above with two half screens (see attached). Can I create a shortcut for these?

-Tim
<img width="2056" alt="SCR-20230326-jiqv" src="https://user-images.githubusercontent.com/128994745/227792568-c2ac8e56-f071-4b2b-9127-e6e0ca8114d7.png">

**Comment by rxhanson** [comment]
Here are the custom shortcuts that would correspond to that layout:

![image](https://user-images.githubusercontent.com/13651296/227793474-d5f6877e-9841-454f-9abc-e86851b6c146.png)

It's currently fairly tedious to build this out by hand at this time and I will be rolling out some updates in an upcoming release to make all of this easier. 

If you're only using this display, you could set up all your windows and let the app use the absolute coordinates for each of these instead of manually typing in the values.

**Comment by CuriousTimmy** [comment]
Thank you for the quick response. I've got the bottom three panels all configured using ninths. It's the top two that I don't know how to configure.

They would be a half in width and 2/3 in height I believe. Could I set that up as well?

It's great to hear you have an update coming out soon to fix this. Any idea when you might release that?

Tim

  On Sun, Mar 26, 2023 at 10:33am, Ryan Hanson ***@***.***> wrote:
Here are the custom shortcuts that would correspond to that layout:

It's currently fairly tedious to build this out by hand at this time and I will be rolling out some updates in an upcoming release to make all of this easier.
If you're only using this display, you could set up all your windows and let the app use the absolute coordinates for each of these instead of manually typing in the values.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"url": "https://github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by CuriousTimmy** [comment]
So sorry. I was looking at this on my phone before and I missed the fact that you mapped all the quadrants out for me. Thank you so much. Let me give this a try.Once I do this will they be available for snapping and window shortcuts as well?-TimOn Sun, Mar 26, 2023 at 10:33am, Ryan Hanson ***@***.***> wrote:Here are the custom shortcuts that would correspond to that layout:It's currently fairly tedious to build this out by hand at this time and I will be rolling out some updates in an upcoming release to make all of this easier.If you're only using this display, you could set up all your windows and let the app use the absolute coordinates for each of these instead of manually typing in the values.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>[
{
***@***.***": "schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"url": "github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "github.com"
}
}
]

**Comment by CuriousTimmy** [comment]
Ryan-Thanks again for the quick reply. It's working great.Do you mind if I ask what screen capture tool you're using. Looks slick with the isolated window and the drop shadow.-TimOn Sun, Mar 26, 2023 at 10:33am, Ryan Hanson ***@***.***> wrote:Here are the custom shortcuts that would correspond to that layout:It's currently fairly tedious to build this out by hand at this time and I will be rolling out some updates in an upcoming release to make all of this easier.If you're only using this display, you could set up all your windows and let the app use the absolute coordinates for each of these instead of manually typing in the values.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>[
{
***@***.***": "schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"url": "github.com/rxhanson/RectanglePro-Community/discussions/246#discussioncomment-5433180",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "github.com"
}
}
]

**Comment by rxhanson** [comment]
cmd+shift+4 space

:)
